src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/types.ts

```diff
@@ -93,3 +93,38 @@ export type ThorchainSaversDepositQuoteResponse =
 export type ThorchainSaversWithdrawQuoteResponse =
   | ThorchainSaversWithdrawQuoteResponseSuccess
   | ThorchainSaversCommonQuoteResponseError
+
+export type ThorchainRunepoolInformationResponseSuccess = {
+  pol: {
+    rune_deposited: string
+    rune_withdrawn: string
+    value: string
+    pnl: string
+    current_deposit: string
+  }
+  providers: {
+    units: string
+    pending_units: string
+    pending_rune: string
+    value: string
+    pnl: string
+    current_deposit: string
+  }
+  reserve: {
+    units: string
+    value: string
+    pnl: string
+    current_deposit: string
+  }
+}
+
+export type ThorchainRunepoolProviderResponseSuccess = {
```

Comment:
same q here re: error-handling

Reply:
Same, axios should throw a natural error in case it doesn't fetch, and it should not error